### PR TITLE
fix: Does not correctly generate pb.goclay.go with filed option nullable = false

### PR DESCRIPTION
**Pull request by mrfoe7**
Information about problem from https://github.com/utrack/clay/issues/93

### CODE REVIEW

File: cmd/protoc-gen-goclay/genhandler/template.go [hunk 1]
```diff
@@ -307,7 +305,12 @@ var (
 					panic(err)
 				}
 
-				ret = append(ret, aExpr.AssignableExpr(assignExpr)+" = &"+fMsg.GoType(goPkg)+"{}")
+				t := fMsg.GoType(goPkg)
+				// nullable option = 65001:0
+				if f.Target.FieldDescriptorProto.GetOptions().String() != "65001:0" {
```
Comment (bullgare) [inline cmd/protoc-gen-goclay/genhandler/template.go:310]: What do you think about moving this magic string to a constant?

**Comment by bullgare** [comment]
BTW, is it possible to add a couple of unit-tests?

File: cmd/protoc-gen-goclay/genhandler/template.go (hunk 1 above)
Comment (mrfoe7) [reply cmd/protoc-gen-goclay/genhandler/template.go]: It is good idea. Done

**Comment by mrfoe7** [comment]
i don't know how to cover this cases by unit testing. Anyone, has idea?

**Comment by utrack** [comment]
You could create a new test set under `/integration`, there's lots of other tests you can use as an example.

**Comment by utrack** [comment]
@mrfoe7 ping

**Comment by mrfoe7** [comment]
@utrack pong :D
Sorry for long wait by my answer. I have a big ping

**Comment by mrfoe7** [comment]
@utrack this problem is in travis-ci and pre build ci.we load protoc-gen-goclay in make script from master, but not from new pull request branch. see next file - `integration/env.mk` in 33 row
```cmd
GOBIN=$(LOCAL_BIN) go install github.com/utrack/clay/v2/cmd/protoc-gen-goclay
```

**Review by utrack — APPROVED** [review]

File: cmd/protoc-gen-goclay/genhandler/template.go [hunk 2]
```diff
@@ -307,7 +309,11 @@ var (
 					panic(err)
 				}
 
-				ret = append(ret, aExpr.AssignableExpr(assignExpr)+" = &"+fMsg.GoType(goPkg)+"{}")
+				t := fMsg.GoType(goPkg)
+				if f.Target.FieldDescriptorProto.GetOptions().String() != nullableOption {
```
[the remark this answers is not in the document]
Comment (mrfoe7) [reply cmd/protoc-gen-goclay/genhandler/template.go]: it is old code version (

**Pull request merged**